dev/sg/cloudsqlproxy/binary.go

```diff
@@ -33,9 +33,21 @@ func Init(download bool) error {
 	}
 	_, err = os.Stat(cloudSQLProxyPath)
 	if err != nil && os.IsNotExist(err) {
-		std.Out.WriteWarningf("cloud-sql-proxy binary not found at %q. try running again with '-download' flag",
-			cloudSQLProxyPath)
-		return errors.Wrapf(err, "failed to find binary")
+		std.Out.WriteWarningf("cloud-sql-proxy binary not found at %q", cloudSQLProxyPath)
+
+		std.Out.Promptf("Would you like me to install cloud-sql-proxy for you?")
```

Comment:
nit: Might be good to include in the prompt the expected input; as obvious as it is 😂 
```suggestion
		std.Out.Promptf("Would you like me to install cloud-sql-proxy for you? [y/N]")
```

Reply:
Nice, thanks!